fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/QueryUtil.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
+import com.ibm.fhir.database.utils.query.expression.StringStatementRenderer;
+import com.ibm.fhir.database.utils.query.node.BindMarkerNode;
+
+/**
+ *
+ */
```

Comment:
What does this class do?

Reply:
It's not obvious ;-)

Reply:
fixed

Reply:
Thanks - it was more to ensure we have a reference as to when the use the pattern